lib/index.ts

```diff
@@ -132,6 +132,19 @@ export class CanvasTable
         };
     }
 
+    private getCellValue(value: CTCellData): string[]
```

Comment:
```ts
private cellValue(data?: CTCellData | string): string
{
        const value = (typeof data === "object" ? data.value : data) || "";
        return value.split("\n").at(0) || "";
}
```

and then use this as
```ts
const cellValue = this.cellValue(row[cellIndex]);
```

Reply:
from what I see, cellValue should return an array of string